test/specs/homeworkOne.js

```diff
@@ -0,0 +1,175 @@
+// Due date: Aug 16 eod
+const{expect} = require("chai");
+/**
+ * Tc-1:
+ * https://www.darsky.net
+ * Verify feelsLikeTempValue is between lowTempValue and highTempValue
+ */
+describe('Homework tests', () => {
+    it('Verify feelsLikeTempValue is between lowTempValue and highTempValue', async () => {
+        /**
+        * Given I am on darksky.net
+        *   And I get feels-like-temp
+        *   And I get low-temp
+        *   And I get high-temp
+        *   And I compare values
+        * Then I verify if feels-like-temp is between low-temp and high-temp
+        */
+        await browser.url('https://darksky.net');
+        await browser.pause(3000);
+
+        const feelsLikeTemp = await $('span[class=feels-like-text]');
+        const highTemp = await $('span[class=high-temp-text]');
+        const lowTemp = await $('span[class=low-temp-text]');
+        
+        expect((feelsLikeTemp >= lowTemp && feelsLikeTemp <= highTemp), 'FeelsLikeTemp is not between high and low temp').to.be.true;
+        
```

Comment:
You supposed to use getText() to get text for comparison. Now you comparing web element not temp.